Source/JavaScriptCore/wasm/WasmBBQJIT64.cpp

```diff
@@ -2966,8 +2969,7 @@ void BBQJIT::emitCatchImpl(ControlData& dataCatch, const TypeDefinition& excepti
     dataCatch.startBlock(*this, emptyStack);
 
     if (exceptionSignature.as<FunctionSignature>()->argumentCount()) {
-        ScratchScope<1, 0> scratches(*this);
-        GPRReg bufferGPR = scratches.gpr(0);
+        GPRReg bufferGPR = wasmScratchGPR;
```

Comment:
I like using `wasmScratchGPR` directly instead of naming it to bufferGPR since it makes using wasmScratchGPR explicit.